app/bundles/LeadBundle/Entity/LeadRepository.php

```diff
@@ -1088,6 +1088,33 @@ public function getLeadOwner($ownerId)
         return $result;
     }
 
+    /**
+     * Check Lead segments by ids.
+     *
+     * @param array<integer> $stages
+     */
+    public function checkLeadStage(Lead $lead, array $stages = []): bool
```

Comment:
Recently we had several discussions in the team whether "in" means:
1.  `stage A` AND `stage B`
2. `stage A` OR `stage B`
Can you make it clear in the method name? Perhaps something like
```suggestion
    public function isContactInOneOfStages(Lead $lead, array $stages = []): bool
```
So it's clear what it does?